bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/config/BigtableOptions.java

```diff
@@ -223,18 +218,22 @@ private BigtableOptions(
     this.clusterAdminHost = Preconditions.checkNotNull(clusterAdminHost);
     this.dataHost = Preconditions.checkNotNull(dataHost);
     this.port = port;
-    this.credential = credential;
     this.projectId = projectId;
-    this.callTimingReportPath = callTimingReportPath;
-    this.callStatusReportPath = callStatusReportPath;
     this.zone = zone;
     this.cluster = cluster;
-    this.retryOptions = retryOptions;
-    this.rpcRetryExecutorService = rpcRetryExecutorService;
     this.customEventLoopGroup = customEventLoopGroup;
     this.channelCount = channelCount;
-    this.timeoutMs = timeoutMs;
-    this.userAgent = userAgent;
+
+    ChannelOptions.Builder optionsBuilder = new ChannelOptions.Builder();
```

Comment:
since this class is called BigtableOptions, call optionsBuilder channelOptionsBuilder, to stress the difference?